module.go

```diff
@@ -94,12 +95,12 @@ func (g *fngen) generate() lua.LGFunction {
 		switch len(out) {
 		case 1:
 			if err := out[0]; !err.IsNil() {
-				state.RaiseError(err.String())
+				state.RaiseError(err.Interface().(error).Error())
```

Comment:
We may need to check if `Interface()` is actually of `error` type.

Reply:
Good point Roman, I thought about it and removed it later
`&& err.Type().Implements(errorInterface)` the point being we do validation when we register a function, don't you think it's like doing it twice 🤔

Reply:
the validation guarantees that the return type is error, no? 🤔

Reply:
doing it [here](https://github.com/kelindar/lua/blob/master/module.go#L155)